packages/qttools.rb

```diff
@@ -3,34 +3,35 @@
 class Qttools < Package
   description 'Qt Tools'
   homepage 'https://github.com/qt/qttools'
-  version '5.15.1'
-  compatibility 'aarch64,armv7l,x86_64'
-  source_url 'http://download.qt.io/official_releases/qt/5.15/5.15.1/submodules/qttools-everywhere-src-5.15.1.tar.xz'
-  source_sha256 'c98ee5f0f980bf68cbf0c94d62434816a92441733de50bd9adbe9b9055f03498'
+  version '5.15.2'
+  compatibility 'all'
+  source_url 'http://download.qt.io/official_releases/qt/5.15/5.15.2/submodules/qttools-everywhere-src-5.15.2.tar.xz'
+  source_sha256 'c189d0ce1ff7c739db9a3ace52ac3e24cb8fd6dbf234e49f075249b38f43c1cc'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.1-chromeos-armv7l.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.1-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/qttools-5.15.2-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: 'b800130b6db80652fc3c9bde2008b8fa64b1e5bab5dfe717c10b8fb04d686b94',
-     armv7l: 'b800130b6db80652fc3c9bde2008b8fa64b1e5bab5dfe717c10b8fb04d686b94',
-     x86_64: 'ca2818363c965077ae2eaf6430369f5a37f5288dd229bbf352072c2a8adffa03',
+  binary_sha256({
+    aarch64: '0d40cb342b8b31427bc976979dbc4c5bfcd8995e5bcc1e74a2a91cf4d84c2265',
+     armv7l: '0d40cb342b8b31427bc976979dbc4c5bfcd8995e5bcc1e74a2a91cf4d84c2265',
+       i686: '1f94a4b40c8f94d47b13445fd9dfb7abb50694cf0876d83e71287f59f36caeca',
+     x86_64: '4fe6c1fcd3b4bdfbdface915f9c7d71b6c73deadbb5bb20cb5d89af8d6d9d2d2'
   })
 
   depends_on 'qtbase'
   depends_on 'libtinfo'
-  depends_on 'llvm'
-  depends_on 'zstd'
-  depends_on 'sommelier'
 
   def self.build
-    system 'qmake && make'
+    # Note: This MAY NOT COMPILE with llvm installed on x86. Install dependencies,
+    # disable depends, and uninstall llvm before attempting compile.
+    system 'qmake -Wall -early QMAKE_CFLAGS=-flto -early QMAKE_CXXFLAGS=-flto && make'
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
+    FileUtils.mkdir_p CREW_DEST_LIB_PREFIX.to_s
```

Comment:
No need to convert to string.

Reply:
I need to find the rule that rubocop uses to lint these files and do this and disable that rule.